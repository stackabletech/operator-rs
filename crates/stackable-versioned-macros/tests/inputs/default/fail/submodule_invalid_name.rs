use stackable_versioned_macros::versioned;

fn main() {
    #[versioned(version(name = "v1alpha1"))]
    mod versioned {
        mod v1alpha2 {}
    }
}
