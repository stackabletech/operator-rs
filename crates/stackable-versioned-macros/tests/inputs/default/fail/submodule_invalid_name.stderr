error: submodules must use names which are defined as `version`s
 --> tests/inputs/default/fail/submodule_invalid_name.rs:6:9
  |
6 |         mod v1alpha2 {}
  |         ^^^
